### DISCUSSION TITLE
Github action generating README.md from imports

### DISCUSSION THREAD

**Discussion by younesStrittmatter** [AutoResearch/autora#386] [Show and Tell]
To simplify and eliminate redundancy in the README.md file, I've developed a GitHub Action that enables you to import markdown files directly from your repository. This is particularly useful for cases like importing index.md and quick_start.md from your docs folder into your repository's README.md file, without having to manually copy them.

You can find an example of how to use this GitHub Action in the following repository: https://github.com/younesStrittmatter/mkdocs-test-child. This setup includes a README-src.md file, where you can write your regular README content and use special tags to import other markdown files. The GitHub Action (located in .github/workflows/generate_readme.yml) will then generate the final README.md file.

Both the source and output file names can be customized in the action, allowing you to adapt it to various projects. If you're interested, I'd be more than happy to provide a quick 5-minute demonstration on how to use this handy GitHub Action. Just let me know!

**Comment by musslick** [comment]
Thank you, Younes! Perhaps you could lead a demo at the next AER maintenance meeting on Tuesday?
